scc/blue-green-with-packageinstall.hbs.md

```diff
@@ -289,19 +277,19 @@ looks similar to the following:
         virtualhost:
           fqdn: www.hello-app.mycompany.com
         routes:
-        - conditions:
-          - prefix: /
-          services:
-          - name: hello-app
-            port: 8080
-            weight: 100
-  ```
-
-1. Update your PackageInstall to include the
+          - conditions:
+            - prefix: /
+            services:
+              - name: hello-app # note the name is changed back
```

Comment:
Is there a guaranteed 0 downtime during this operation? I see that the current approach is for the rename of the green deployment is happening simultaneously as the rerouting of traffic. How does kapp-controller guarantee the order of operations?

Reply:
when I tried there was no down time for me